src/ankisyncd/full_sync.py

```diff
@@ -23,7 +23,7 @@ def test_db(self, db: DB):
                 "Integrity check failed for uploaded collection database file."
             )
 
-    def upload(self, col: Collection, data: bytes, session) -> str:
+    def upload(self, col: Collection, data: bytes, session) :
```

Comment:
What is the benefit of removing this type hint?

Reply:
this may be caused by mis-operation and missed part has already been added.